src/app/shared/components/advertising-area/advertising-area.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-advertising-area',
+  templateUrl: './advertising-area.component.html',
+  styleUrls: ['./advertising-area.component.scss']
+})
+export class AdvertisingAreaComponent implements OnInit {
+
+  @Input() contentHeight: number;
+  @Input() contentWidth: number;
+  content: string;
```

Comment:
1. content is also input property. It should come from attribute
2. Rename contentHeight and contentWidth to just **height** and **width** and make them optional, because they may not exist. The required attribute only content